packages/driver/src/cy/logGroup.ts

```diff
@@ -27,6 +27,7 @@ export default (Cypress, userOptions: Cypress.LogGroup.Config, fn: Cypress.LogGr
   const endLogGroupCmd = $Command.create({
     name: 'end-logGroup',
     injected: true,
+    args: [],
```

Comment:
what's this change for?

Reply:
Consistency. This homogenizes `$Commands` created here with commands created everywhere else in Cypress. Adding the empty array here means that elsewhere in the PR I can assume that all commands have an args array.

I've honestly forgotten exactly where it was throwing an error, but it was somewhere pretty arcane, and this one-line change meant I didn't have to add existence checks deep inside the command-queue.